packages/memory-storage/src/background-handler/index.ts

```diff
@@ -0,0 +1,35 @@
+import { randomUUID } from 'node:crypto';
+
+import { handleMessage } from './fs-utils';
+import type { BackgroundHandlerReceivedMessage } from '../utils';
+
+export const promiseMap: Map<string, {
```

Comment:
why do you need this? maybe add some comment

Reply:
I'll add that comment when back, but its purpose is so we can then wait for all fs operations to finish before the process ends